ecs/elections/exceptions.py

```diff
@@ -15,12 +15,12 @@ def __str__(self):
 
 class IncorrectPreferenceOrderException(Exception):
     def __str__(self):
-        return "Error of votes' preference order"
+        return "Error of votes preference order"
```

Comment:
but preference is rather the trait of a voter and can be used imho as a synonim of vote (as the sequence of candidates)